tensorflow_addons/optimizers/weight_decay_optimizers.py

```diff
@@ -21,6 +21,14 @@
 from typing import Union, Callable, Type
 
 
+def _ref(var):
```

Comment:
tf.addons supports TF2 only anyway, maybe we can remove this string comparison and just return var.ref()/experimental_ref()?

Reply:
Done.